src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -230,7 +230,8 @@ private function executeAuthenticator(AuthenticatorInterface $authenticator, Req
     private function handleAuthenticationSuccess(TokenInterface $authenticatedToken, PassportInterface $passport, Request $request, AuthenticatorInterface $authenticator): ?Response
     {
         // @deprecated since 5.3
-        if (!method_exists($authenticatedToken->getUser(), 'getUserIdentifier')) {
+        $user = $authenticatedToken->getUser();
+        if (\is_object($user) && $user instanceof UserInterface && !method_exists($user, 'getUserIdentifier')) {
```

Comment:
`is_object($user)` is useless here. `$user instanceof UserInterface` won't ever be true for non-objects

Reply:
`TokenInterface::getUser()` might also return `Stringable`

Reply:
Oh yea, nevermind, of course

Reply:
Done